zellij-server/src/screen.rs

```diff
@@ -148,6 +150,7 @@ pub(crate) struct Screen {
     position_and_size: PositionAndSize,
     /// The index of this [`Screen`]'s active [`Tab`].
     active_tab_index: Option<usize>,
+    last_active_tab_index: Option<usize>,
```

Comment:
I think `previous_active_tab_index` would be clearer here, last could be a little ambiguous,
what do you think?

Reply:
Done.